great_expectations/render/renderer_configuration.py

```diff
@@ -590,34 +577,22 @@ def add_param(
                 param_types=param_type, value=value
             )
 
-        renderer_params_args: Dict[str, Optional[Any]]
         if value is None:
-            renderer_params_args = {
-                **self.params.dict(exclude_none=False),
-                name: None,
-            }
+            self.params.__dict__[name] = None
+            return
+
+        assert isinstance(param_type, RendererValueType)
+
+        # if we already moved the suite parameter raw_kwargs to a param,
+        # we need to combine the param passed to add_param() with those existing raw_kwargs
+        if name in self.params.__dict__ and self.params.__dict__[name]["suite_parameter"]:
+            self.params.__dict__[name] = renderer_param(
+                schema=RendererSchema(type=param_type),
+                value=value,
+                suite_parameter=self.params.__dict__[name]["suite_parameter"],
+            )
```

Comment:
Can you do this with standard dict unpacking?
```python
original = {"a": 1, "b": 2}
update = {"a": 2, "c": 3}
merged = {**original, **update}
```

Note: there are several different ways to merge dictionaries, ☝️ this method is generally the best.
https://treyhunner.com/2016/02/how-to-merge-dictionaries-in-python/

Reply:
I'm using a generic example of merging dictionaries.
I didn't mean to suggest that you create 3 new dictionaries.